src/index.html

```diff
@@ -7,12 +7,69 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100..900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-container"></div>
+
+      <div class="card__description">
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__id">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--1"></div>
+
+            <div class="stars__star stars__star--2"></div>
+
+            <div class="stars__star stars__star--3"></div>
+
+            <div class="stars__star stars__star--4"></div>
+
+            <div class="stars__star stars__star--5"></div>
```

Comment:
```suggestion
          <div class="stars stars--1">
            <div class="stars__star"></div>

            <div class="stars__star"></div>

            <div class="stars__star"></div>

            <div class="stars__star"></div>

            <div class="stars__star"></div>
```
Use the same logic as in the previous task. You should show active stars depending on the modifier